spec/models/alchemical_property_spec.rb

```diff
@@ -6,32 +6,43 @@
   describe 'validations' do
     describe 'name' do
       it 'must be present' do
-        property = described_class.new
+        model = build(:alchemical_property, name: nil)
```

Comment:
This is the style of the other spec files, so I've made some changes to this one to bring it in line.